tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -421,6 +421,22 @@ We're finally ready to run the project:
 
 .. image:: img/gdextension_cpp_animated.gif
 
+Custom editor icon
+------------------
+By default, Godot uses the node icon in the scene dock. The custom icon can be
+added via the ``gdextension`` file. Write a line where the name of the node
+class equals the resource path to the icon's SVG file.
```

Comment:
```suggestion
By default, Godot uses the node icon it is inheriting from in the scene dock. The custom icon can be
added via the ``gdextension`` file.  The node's icon is set by reference to its name and resource path
of an SVG file.
```

Reply:
Ummm my editor is not using the icon node it is inheriting from. Is it a bug?

![Bug](https://github.com/godotengine/godot-docs/assets/2855916/5e38d2c4-0e97-4ec0-ab7b-45347a04a43b)

Reply:
In that Case I am wrong. Then leave out the inheriting part.